WordPress/Classes/ViewRelated/Tools/SettingsListPickerViewController.swift

```diff
@@ -0,0 +1,160 @@
+import Foundation
+import WordPressShared
+
+
+
+/// SettingsListPicker will render a list of options, and will allow the user to select one from the list.
+///
+class SettingsListPickerViewController : UITableViewController
+{
+    /// Header Strings to be applied over the diferent sections
+    ///
+    var headers : [String]?
+
+    /// Footer Strings to be applied over the diferent sections
+    ///
+    var footers : [String]?
+    
+    /// Titles to be rendered
+    ///
+    var titles : [[String]]?
+
+    /// Row Subtitles. Should contain the exact same number as titles
+    ///
+    var subtitles : [[String]]?
```

Comment:
Instead of relying on the developer to provide matching arrays why not make these properties privates, and create a setter method, that checks for these conditions?